tensorrt_llm/commands/serve.py

```
@@ -169,8 +169,11 @@ def launch_server(host: str,
         # AutoDeploy does not support cache reuse yet.
         llm_args["kv_cache_config"].enable_block_reuse = False
         llm = AutoDeployLLM(**llm_args)
-    else:
+    elif backend == 'tensorrt':
         llm = LLM(**llm_args)
+    else:
+        raise ValueError(
+            f"Invalid backend: {backend}, check help for available options.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)